src/views/List.jsx

```diff
@@ -1,11 +1,21 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	/**
+	 * Retrieve the user's shopping list token from the local storage.
+	 * This is a temporary solution to display the token on the page,
+	 * ensuring that users have a reference in case they forget their unique token.
+	 */
+	const token = localStorage.getItem('tcl-shopping-list-token');
```

Comment:
One optional suggestion here is to pass `listToken` as a prop from App.jsx so you don't have to create a new token on line 9